<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>404</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>361</width>
    <height>404</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>BMI-Rechner</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ressourcen.qrc">
    <normaloff>:/icon/personenwaage.png</normaloff>:/icon/personenwaage.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="labelTitel">
        <property name="minimumSize">
         <size>
          <width>171</width>
          <height>31</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 255);</string>
        </property>
        <property name="text">
         <string>Body Mass Index</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Körpergröße (cm):</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spinBoxGroesse">
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="minimum">
           <number>34</number>
          </property>
          <property name="maximum">
           <number>240</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Gewicht (kg):</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QDoubleSpinBox" name="doubleSpinBoxGewicht">
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="minimum">
           <double>15.000000000000000</double>
          </property>
          <property name="maximum">
           <double>170.990000000000009</double>
          </property>
          <property name="singleStep">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Alter:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="spinBoxAlter">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="accelerated">
           <bool>true</bool>
          </property>
          <property name="showGroupSeparator" stdset="0">
           <bool>true</bool>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Geschlecht:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="radioButtonMann">
          <property name="text">
           <string>männlich</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="radioButtonWeib">
          <property name="text">
           <string>weiblich</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBoxAmputation">
        <property name="text">
         <string>Amputation:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboBoxAmputation">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <item>
         <property name="text">
          <string>Körperteil</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hand</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Unterarm</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Oberarm</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Fuß</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Unterschenkel</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Oberschenkel</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="checkBoxBeidseitig">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>beidseitig</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 255);</string>
        </property>
        <property name="text">
         <string>BMI:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="labelBMI">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btnAuswertungFarbe">
        <property name="minimumSize">
         <size>
          <width>31</width>
          <height>31</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>31</width>
          <height>31</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="ressourcen.qrc">
          <normaloff>:/icon/leichtes-uebergewicht.png</normaloff>:/icon/leichtes-uebergewicht.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="labelAuswertung">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>leichtes Übergewicht</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>62</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="labelBewertung">
        <property name="minimumSize">
         <size>
          <width>91</width>
          <height>81</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>91</width>
          <height>81</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="ressourcen.qrc">:/icon/daumen-normgewicht.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPlainTextEdit" name="plainTextEditHinweis">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="overwriteMode">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>361</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuProgramm">
    <property name="title">
     <string>Programm</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionBerechnen"/>
    <addaction name="actionWerte_zuruecksetzen"/>
    <addaction name="actionProgramm_beenden"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHilfe">
    <property name="title">
     <string>Hilfe</string>
    </property>
    <addaction name="action_ueber_das_Programm"/>
   </widget>
   <addaction name="menuProgramm"/>
   <addaction name="menuHilfe"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBerechnen_TB"/>
   <addaction name="actionWerte_zuruecksetzen_TB"/>
   <addaction name="separator"/>
   <addaction name="action_ueber_das_Programm_TB"/>
   <addaction name="separator"/>
   <addaction name="actionProgramm_beenden_TB"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionWerte_zuruecksetzen">
   <property name="icon">
    <iconset resource="ressourcen.qrc">
     <normaloff>:/icon/reset.png</normaloff>:/icon/reset.png</iconset>
   </property>
   <property name="text">
    <string>Werte zurücksetzen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionBerechnen">
   <property name="icon">
    <iconset resource="ressourcen.qrc">
     <normaloff>:/icon/berechnen.png</normaloff>:/icon/berechnen.png</iconset>
   </property>
   <property name="text">
    <string>Berechnen</string>
   </property>
   <property name="shortcut">
    <string>Alt+B</string>
   </property>
  </action>
  <action name="actionProgramm_beenden">
   <property name="icon">
    <iconset resource="ressourcen.qrc">
     <normaloff>:/icon/ende.png</normaloff>:/icon/ende.png</iconset>
   </property>
   <property name="text">
    <string>Programm beenden</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_ueber_das_Programm">
   <property name="icon">
    <iconset resource="ressourcen.qrc">
     <normaloff>:/icon/about.png</normaloff>:/icon/about.png</iconset>
   </property>
   <property name="text">
    <string>Über das Programm</string>
   </property>
   <property name="shortcut">
    <string>Alt+A</string>
   </property>
  </action>
  <action name="action_ueber_das_Programm_TB">
   <property name="icon">
    <iconset resource="ressourcen.qrc">
     <normaloff>:/icon/about.png</normaloff>:/icon/about.png</iconset>
   </property>
   <property name="text">
    <string>Über das Programm</string>
   </property>
  </action>
  <action name="actionWerte_zuruecksetzen_TB">
   <property name="icon">
    <iconset resource="ressourcen.qrc">
     <normaloff>:/icon/reset.png</normaloff>:/icon/reset.png</iconset>
   </property>
   <property name="text">
    <string>Werte zurücksetzen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionBerechnen_TB">
   <property name="icon">
    <iconset resource="ressourcen.qrc">
     <normaloff>:/icon/berechnen.png</normaloff>:/icon/berechnen.png</iconset>
   </property>
   <property name="text">
    <string>Berechnen</string>
   </property>
  </action>
  <action name="actionProgramm_beenden_TB">
   <property name="icon">
    <iconset resource="ressourcen.qrc">
     <normaloff>:/icon/ende.png</normaloff>:/icon/ende.png</iconset>
   </property>
   <property name="text">
    <string>Programm beenden</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ressourcen.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionBerechnen_TB</sender>
   <signal>triggered()</signal>
   <receiver>actionBerechnen</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionProgramm_beenden_TB</sender>
   <signal>triggered()</signal>
   <receiver>actionProgramm_beenden</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_ueber_das_Programm_TB</sender>
   <signal>triggered()</signal>
   <receiver>action_ueber_das_Programm</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionWerte_zuruecksetzen_TB</sender>
   <signal>triggered()</signal>
   <receiver>actionWerte_zuruecksetzen</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxAmputation</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBoxAmputation</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxAmputation</sender>
   <signal>toggled(bool)</signal>
   <receiver>checkBoxBeidseitig</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>306</x>
     <y>222</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
